lib/once.js

```diff
@@ -1,9 +1,9 @@
 import on from './on';
 import off from './off';
 
-function once(el, event, run, capture = false) {
-  on(el, event, e => {
-    off(el, event);
+function once(selector, event, run, capture = false) {
+  on(selector, event, e => {
+    off(selector, event);
```

Comment:
`off()` doesn't receive a capture option since `removeEventListener()` doesn't have one. The docs have this `off(selector, event)`